util/cron/publish-docker-images.bash

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# This script will build the docker images for 'amd' and 'arm' platforms using buildx and publish the images to the docker registry
+
+# Check if the script is run with correct arguemnrt if not fail
+if [  $# -ne 2  ]
+then
+    echo " Docker repository and/or image-version not supplied "
+    echo " run with ./publish-docker-images.bash doc_repository image_version "
+    exit 1
+fi
+echo "RepositoryName: $1"
+echo "imageVersion: $2"
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common.bash
+source $CWD/docker.bash
+export CHPL_HOME=$(cd $CWD/../.. ; pwd)
+log_info "Setting CHPL_HOME to: ${CHPL_HOME}"
+
+start_docker
+ 
+# build_publish will build multi platform chapel docker images, tags them, and pushes the images to the docker repository .
+
+build_publish(){
+
+local registry="$1"
+local imageName="$2" 
+local version="$3"
+
+# the below buildx command will build images for amd and arm, tags with the tags specified, and pushes it to the docker repository($registry)
+docker buildx build --platform=linux/amd64,linux/arm64 . --push -t $registry/$imageName:$version -t  $registry/$imageName:latest
+
+    if [ $PUBLISHED -ne 0 ]
```

Comment:
looks like this might have been indented by accident

Reply:
Fixed.